README.md

```diff
@@ -0,0 +1,32 @@
+
+| Package          | Lint                           | Build                           | Test                                                                                     |
+|------------------|--------------------------------|---------------------------------|------------------------------------------------------------------------------------------|
+| catbuffer/parser | ![lint][catbuffer-parser-lint] |                                 | ![test][catbuffer-parser-test]                                                           |
+|                  |                                |                                 |                                                                                          |
+| client/rest      | ![lint][client-rest-lint]      |                                 | ![test][client-rest-test]                                                                |
+|                  |                                |                                 |                                                                                          |
+| sdk/javascript   | ![lint][sdk-javascript-lint]   |                                 | ![test][sdk-javascript-test] ![vectors][sdk-javascript-vectors]                          |
+| sdk/python       | ![lint][sdk-python-lint]       | ![build][sdk-python-build]      | ![test][sdk-python-test] ![examples][sdk-python-examples] ![vectors][sdk-python-vectors] |
+|                  |                                |                                 |                                                                                          |
+| jenkins          | ![lint][jenkins-lint]          |                                 |                                                                                          |
+| linters          | ![lint][linters-lint]          |                                 |                                                                                          |
+
+[catbuffer-parser-lint]: https://jenkins.symboldev.com/buildStatus/icon?job=Symbol%2Ftest%20folder%2FParser%2Fjenkins_symbol_files%2F&config=catbuffer-parser-lint
```

Comment:
1. what (if anything) should this link to?
2. what branch should we display? dev? main?